src/main/java/org/eclipse/tractusx/autosetup/testservice/proxy/ConnectorTestRequest.java

```diff
@@ -0,0 +1,16 @@
+package org.eclipse.tractusx.autosetup.testservice.proxy;
+
+import lombok.Builder;
+import lombok.Data;
```

Comment:
FYI: https://github.com/eclipse-tractusx/tractusx-edc/pull/143 perhaps something to discuss in your team

Reply:
The impact is high, We will discuss this with @Mehran Roshandel.